.babelrc

```diff
@@ -61,17 +61,6 @@
         "loose": true
       }
     ],
-    [
-      // This plugin is configured by preset-env when needed... however because
-      // Webpack v4 doesn't understand this new syntaxe we need to
-      // unconditionally transpile it even for platforms that don't support it.
-      "@babel/plugin-proposal-nullish-coalescing-operator",
```

Comment:
Looks like we still have `@babel/plugin-proposal-nullish-coalescing-operator` dependency in the package.json file. It seems like it's included in the preset already, so we don't have to explicitly add it. It could be good to remove it from package.json.

Reply:
ah right, I can indeed remove it from package.json!
Note that while it's included in the preset but as explained in the comment in our case it wasn't used because of our targets... but we wanted to use it because webpack 4 wouldn't understand the syntax.